frontend/src/components/Profile.js

```diff
@@ -116,6 +120,22 @@ class Profile extends React.Component {
     
   }
 
+  getNamesOfAttendees(refAttendees) {
+    var names = []
+    var len = refAttendees.length;
+    var ref = fb.userRef.child('users');
+
+    for (var i = 0; i < len; i++) {
+      var current_attendee = ref.child(refAttendees[i]).child('name');
+
+      current_attendee.on('value', snapshot => {
+        let name = snapshot.val();
+        names.push(name);
```

Comment:
not super familiar with firebase, is this done synchronously? Will `names` contain all the attendee's names when it's returned?